modules/repository-datastax/src/test/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepositoryTest.java

```diff
@@ -722,4 +723,31 @@ public void testGetTagList() throws ExecutionException, InterruptedException {
         assertArrayEquals(exp2.toArray(), callResult.get(app02).toArray());
     }
 
+
+    @Test
+    public void testGetEmptyTagList() throws ExecutionException, InterruptedException {
+        //------ Input --------
+        Application.Name app01 = Application.Name.valueOf("app01");
```

Comment:
@AndreaSuckro There is only 1 app, 1 experiment and 1 db result in this test. Let's do away with the number identifier at the end of each of those variables.

Also, I took a look at the previous test as well (which was checked in earlier - apologies for not suggesting it then)  - Let us try to name variables as meaningfully as we can instead of using numbers at the end of variables. For example `dbResult2` could be changed to `dbResultforapp2` etc. etc.